lib/lighthouse/benefits_documents/configuration.rb

```diff
@@ -205,6 +207,66 @@ def claimant_can_upload_document(document_data)
       connection.post(DOCUMENT_VALIDATE_CLAIMANT_PATH, body, headers)
     end
 
+    # Returns a list of all VBMS document names related to participantId.
+    # @param participant_id: string A unique identifier assigned to each patient entry
+    # in the Master Patient Index linking patients to their records across VA systems.
+    # Example: 999012105
+    # @param page_number: integer 1-based page number to retrieve. Defaults to 1.
+    # Example: 1
+    # @param page_size: integer Number of results per page (1–100). Defaults to 100. Maximum 100.
+    # Example: 100
+    def participant_documents_search(participant_id:, page_number: 1, page_size: 100)
+      headers = { 'Authorization' => "Bearer #{
+        access_token(
+          nil,
+          nil,
+          {}
+        )
+      }" }
+
+      body = {
+        'data' => {
+          'participantId' => participant_id
+        },
+        'pagination' => {
+          pageNumber: page_number,
+          pageSize: page_size
+        }
+      }
+      connection.post(PARTICIPANT_DOCUMENTS_SEARCH_PATH, body, headers)
+    end
+
+    # Download the full content of a document (such as a PDF).
+    # The document must be identified by its unique ID, and associated with either a Participant ID or File Number.
+    # @param document_uuid: string The document's unique identifier in VBMS,
+    # obtained by making a Document Service API request to search for documents
+    # that are available to download for the Veteran.
+    # Note that this differs from the document's current version UUID.
+    # @param participant_id: string A unique identifier assigned to each patient entry
+    # in the Master Patient Index linking patients to their records across VA systems.
+    # Example: 999012105
+    # @param file_number: The Veteran's VBMS fileNumber used when uploading the document to VBMS.
+    # It indicates the eFolder in which the document resides.
+    # Example: 999012105
+    def participant_documents_download(document_uuid: nil, participant_id: nil, file_number: nil)
```

Comment:
The comment above says "The document must be identified by its unique ID, and associated with either a Participant ID or File Number." This to me says that the document_uuid is required every time. 

```
def participant_documents_download(document_uuid:, participant_id: nil, file_number: nil)
  raise ArgumentError, 'document_uuid required' if document_uuid.blank?
  raise ArgumentError, 'participant_id or file_number required' if participant_id.blank? && file_number.blank?
  ...
  ```